### PR TITLE
Add google tag manager snippets

### PR DESCRIPTION
**Pull request by tramuntanal**
This PR adds the snippets for the Google Tag Manager.

### CODE REVIEW

**Review by laurajaime — APPROVED** [review]
LGTM!

**Review by laurajaime — CHANGES_REQUESTED** [review]
Good job!!

Remember add `es.yml` translations to `config/locales/google_tag_manager/`

File: config/locales/google_tag_manager/es.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+es:
+  activemodel:
+    attributes:
+      organization:
+        entity_code: Código de ente
+        entity_name: Nombre de ente
```
Comment (laurajaime) [inline config/locales/google_tag_manager/es.yml:6]: Maybe `entidad` is better, what do you think?

Comment (tramuntanal) [reply config/locales/google_tag_manager/es.yml]: done!

**Review by laurajaime — APPROVED** [review]

**Pull request merged**